plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/models/IRunningPodHolder.java

```diff
@@ -0,0 +1,16 @@
+/*******************************************************************************
+ * Copyright (c) 2016 Red Hat, Inc. Distributed under license by Red Hat, Inc.
+ * All rights reserved. This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution, and is
+ * available at http://www.eclipse.org/legal/epl-v10.html
+ * 
+ * Contributors: Red Hat, Inc.
+ ******************************************************************************/
+package org.jboss.tools.openshift.internal.ui.models;
+
+/**
+ * Running pods may be adapted to this interface with Adapters.adapt().
+ */
+public interface IRunningPodHolder {
+	public IOpenshiftUIElement<?, IOpenshiftUIElement<?,?>> getPodUIElement();
```

Comment:
we could elaborate a more precise interface but then we'd most likely have to refactor quite some bits here and there. Accepting this complaint for now.